<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmImportData</class>
 <widget class="QDialog" name="FrmImportData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbDataPreview">
     <property name="title">
      <string>Data Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tblPreview"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbSeparatorOptions">
     <property name="title">
      <string>Separator options - Separated by</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbOther">
          <property name="text">
           <string>Other</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtOther">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maxLength">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="rbSpace">
        <property name="text">
         <string>Space</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rbSemicolon">
        <property name="text">
         <string>Semicolon</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbComma">
        <property name="text">
         <string>Comma</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbTab">
        <property name="text">
         <string>Tab</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtFile">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>Import</string>
       </property>
       <property name="icon">
        <iconset resource="resursi.qrc">
         <normaloff>:/icons/icons/submit.png</normaloff>:/icons/icons/submit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resursi.qrc"/>
 </resources>
 <connections/>
</ui>
